website/build.gradle

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Define common lifecycle tasks and artifact types
+apply plugin: "base"
+
+task cleanWebsite(type: Delete) {
+  delete '.sass-cache', 'vendor', '.testcontent', 'content'
+}
+clean.dependsOn cleanWebsite
+
+task installBundler(type: Exec) {
+  commandLine './install_bundler.sh'
+}
+
+task installGems(type: Exec) {
+  mustRunAfter cleanWebsite, installBundler
+  inputs.file 'Gemfile.lock'
+  outputs.dir 'vendor'
+  commandLine 'bundle', 'install', '--deployment'
+}
+
+// TODO(BEAM-4696): Remove this once we can run Jenkins jobs within Docker
+task setupJenkins {
+  dependsOn installBundler, installGems
+}
+
+task buildWebsite(type: Exec) {
+  dependsOn installGems
+  mustRunAfter cleanWebsite, setupJenkins
+  inputs.files 'Gemfile.lock', '_config.yml'
+  inputs.dir 'src'
+  outputs.dir '.sass-cache'
```

Comment:
Is there a way to get the outputs to go into the project build directory instead of a the root of the subproject?